guides/customization/vnc.md

```diff
@@ -74,15 +74,22 @@ environment.
 
 1. Click **Create Environment**
 
-## Option 1: Connect via web
+## Connecting to Coder
+
+There are two ways you can connect to your environment:
+
+- Connect via the web
+- Connect using a local VNC client
+
+### Option 1: Connect via web
 
 If your image includes [noVNC](https://github.com/novnc/noVNC), or another
 web-based client, you can use a [dev URL](../../environments/devurls.md) to
 access it securely.
 
 1. From the **environment overview** page, click **Add URL**
 
-1. Provide the **Port** number that noVNC is running on (this information is
+1. Provide the **Port** number that the VNC is running on (this information is
```

Comment:
We should still specify that the user is accessing the VNC _web client_, to differentiate from the VNC _server_ used in the image as well.